client/state/data-getters/index.js

```diff
@@ -101,8 +101,9 @@ export const requestGeoLocation = () =>
 		{ fromApi: () => ( { body: { country_short } } ) => [ [ 'geo', country_short ] ] }
 	);
 
-export const requestSiteAlerts = siteId => {
+export const requestSiteAlerts = ( siteId, force ) => {
```

Comment:
Some other functions in this module have freshness as a parameter, for example:

`export const requestActivityLogs = ( siteId, filter, { freshness = 5 * 60 * 1000 } = {} ) => {`

It doesn't look like they are ever used, but we may as well stick to the same pattern, passing in 0 when we want to force an update.